src/dummyData.ts

```diff
@@ -0,0 +1,217 @@
+// import { InjectionPoint } from 'components/InjectionPoint';
+export interface Province {
+  id: number;
+  name: string;
+}
+export interface District {
+  id: number;
+  name: string;
+  provinceId: number;
+}
+export interface Ward {
+  id: number;
+  name: string;
+  districtId: number;
+}
+
+export interface InjectionPoint {
+  id: number;
+  name: string;
+  detailAddress: string;
+  wardId: number;
+  districtId: number;
+  provinceId: number;
```

Comment:
```suggestion
  wardId: number;
```

Chỉ cần lưu wardId là đủ. từ ward tìm ngược lại province và district